package.json

```diff
@@ -82,7 +82,7 @@
     "@types/gzip-size": "^5.1.1",
     "@types/inquirer": "^0.0.44",
     "@types/jasmine": "^3.3.13",
-    "@types/node": "^12.6.2",
+    "@types/node": "^12.6.2 < 12.12.42",
```

Comment:
This is needed because DefinitelyTyped/DefinitelyTyped#44700, which is published to `@types/node@12.12.42`. Zone.js v0.10.2 has a type definition of `global` that is different to the `@types/node` one. The fix for Zone.js has already been merge (https://github.com/angular/angular/pull/37861), but waiting for 0.11.3 release cut.

I've also tried upgrade Zone.js to 0.11.2 (latest at the time of writing), I've got 2 test failures. So we might need to fix the broken code to be compatible with the new Zone.js to remove this weird constraint.